styleguide/_themes/derek/scss/patterns/lead.scss

```diff
@@ -1,18 +1,18 @@
-.lead-bodyWrapper {
-  margin-bottom: 14px;
```

Comment:
Does this really need to be done then? It doesn't really have anything to do with template documentation, and it just adds to the noise of this PR. I mean I'm not gonna stop the PR just over this one item, but it just seems like an unnecessary step, when we're already doing a lot here. If it was just this one thing, whatever, but there's a lot of these unrelated cleanup items in this PR: Leads, tabs, blog feed, landing pages, etc...  I'm all for cleaning that stuff up, and we can even associate with the same ticket number if you want, but to me it seems like that should be handled in a separate PR. As far as this PR is concerned we should just focus on the template documentation as that is the actual purpose of this ticket, especially since we're not exactly sure when this PR is gonna be merged in relative to the other major zoolander changes that are coming. We don't know which components are going to be modularized at that point and which ones aren't.  The more noise we have, the harder it is going to be to resolve those conflicts correctly.